freedom/llh_service/kill_service.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+"""
+sends a "die" command to the llh service
+causes a running llh service to exit its work loop
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__author__ = "Aaron Fienberg"
+
+import json
+import sys
+
+import zmq
+
+
+def main():
+    with open("service_params.json") as f:
```

Comment:
path is relative to PWD, so this script will fail in most use cases

Reply:
I've moved this to the serviceTests example directory, and made the scripts more general regarding paths.